otherlibs/unix/unix.mli

```diff
@@ -307,6 +307,9 @@ val openfile : string -> open_flag list -> file_perm -> file_descr
 val close : file_descr -> unit
 (** Close a file descriptor. *)
 
+val fsync : file_descr -> unit
+(** Synchronize a file's in-core state with the storage device. *)
```

Comment:
Does `fsync` ever cause anything to be read? Synchronize to me implies both directions: how about "Flushes all in-core data of the file associated with the `file_descr` to the storage device."?

Reply:
(I realise the man page for `fsync` begins by using the word synchronisation - it's just that it then goes on to describe flushing only AFAICT! FlushFileBuffers definitely does only what it says on the tin...)

Reply:
The phrasing was taken straight from the summary line of the man page in *BSD and OS X.

I agree, though, that "flush in-memory file buffers to disk" or some such is better.